spatial/r2/box.go

```diff
@@ -0,0 +1,101 @@
+// Copyright ©2022 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package r2
+
+import "math"
+
+// Box is a 2D bounding box. Well formed
+// Boxes have Min components smaller than
+// Max components.
```

Comment:
Would you mind going through the docs and wrapping at approximately 70-80 unless there are reasons to not.